sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -4497,6 +4500,222 @@ SUBROUTINE CALSLR_UUTAH(SLR)
       END SUBROUTINE CALSLR_UUTAH
 !
 !-------------------------------------------------------------------------------------
+!
+!> Computes snow solid-liquid-ratio slr using the Steenburgh 2024 algorithm.
+!>
+!> Obtained the code and data from U of Utah Jim Steenburgh, 
+!> Peter Veals, and Michael Pletcher.
+!>
+!> @param[out] SLR real Solid snow to liquid ratio
+!> 
+!> ### Program history log:
+!> Date | Programmer | Comments
+!> -----|------------|---------
+!> 2024-11-15 | Jesse Meng | Initial
+!>
+!> @author Jesse Meng @date 2024-11-15
+
+      SUBROUTINE CALSLR_UUTAH2(SLR)
```

Comment:
@JesseMeng-NOAA Can you update this new subroutine to follow the format below?

- all lowercase

- indentation for loops, conditionals and blocks